products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -158,6 +160,8 @@ extension BuildKeyKind: CustomStringConvertible {
             return "filteredDirectoryContents"
         case .stat:
             return "stat"
+        @unknown default:
+            return "unknown"
```

Comment:
This is ambiguous with the one above, can you please use something else, perhaps `"BuildKeyKind(\(rawValue))"`?

Reply:
Not sure what you mean by `"BuildKeyKind(\(rawValue))"`. I can return something like `"<unkown>"` if you want.

Reply:
Done 😄 Thanks for the review!